packages/core/src/components/FilterGroup/RightPanel/RightPanel.tsx

```diff
@@ -49,8 +49,11 @@ export const HvFilterGroupRightPanel = ({
   } = useContext(HvFilterGroupContext);
 
   const activeGroupOptions = useMemo(
```

Comment:
I noticed the following when typing "category":

https://github.com/lumada-design/hv-uikit-react/assets/43220251/87ef810b-b4ae-4081-a3f7-53b220dd65d4

I think this should solve the problem:

```
const activeGroupOptions = useMemo(
    () =>
      filterOptions[activeGroup]?.data
        .filter((option) =>
          option.name.toLowerCase().includes(searchStr.toLowerCase())
        )
        .map((option) => option.id) || [],
    [filterOptions, activeGroup, searchStr]
);
```

* I added `toLowerCase()` and removed the `trim()` since if we type "category " only 3 options are shown but the counter has 4.